service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -123,7 +124,8 @@
     private final List<Element<Placement>> placements;
     private final Order standardDirectionOrder;
 
-    @NotNull(message = "You need to enter the allocated judge.", groups = SealedSDOGroup.class)
+    @NotNull(message = "You need to enter the allocated judge.",
```

Comment:
would prefer something more like "You need to allocate a judge to the case" but it's for content to decide

Reply:
Yep 👍 I've spoke to UX and they'd like a story created to update this wording. Out of scope for this story.